portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro

```diff
@@ -741,7 +741,7 @@ definition {
 		}
 	}
 
-	macro viewFieldReference {
+	macro viewFieldReferenceValueFromSideTab {
```

Comment:
Hi @ewertonluna , is there a specific reason to rename this macro? Because I think it doesn't add anything 
and `viewFieldReference` seems more clean and easy to read than `viewFieldReferenceValueFromSideTab`, but if it is necessary, I would suggest you to rename to Sidebar instead of SideTab, because of the Forms terminology.

Reply:
Hi @mlan0! You're right, I shouldn't have renamed the macro. I made a mistake thinking that it was a Help Text field, which can be located both in the sidebar or in the Form pages, so it could be useful to differentiate. My bad, I'll change it back again. Thank you!